grails-app/domain/org/pih/warehouse/order/Order.groovy

```diff
@@ -518,6 +518,10 @@ class Order implements Serializable {
         return orderType?.isTransferOrder()
     }
 
+    Boolean isFullyInvoiceable() {
+        return orderItems.every{ it -> it.invoiceable }
```

Comment:
I believe you want the order item check here to be invoiceable || has already been invoiced.

If we've previously generated an invoice for some of the order items, those will no longer be invoiceable but shouldn't cause the pop-up if the remaining items are invoiceable

Reply:
you are correct, I made some changes that should reflect this behavior.
I decided to check invoicebale against encumbered items which should make sense, right? 

Additionally added a simple unit test to reinforce this method